actions/bulk_link_issue.py

```diff
@@ -6,16 +8,40 @@
 
 
 class BulkLinkJiraIssueAction(BaseJiraAction):
+    def link_issues(
+        self,
+        semaphore,
+        issue_key=None,
+        target_issue=None,
+        direction=None,
+        link_type=None,
+    ):
+        with semaphore:
+            if direction == "outward":
+                outward_issue_key = issue_key
+                inward_issue_key = target_issue
+                issue = self._client.create_issue_link(
```

Comment:
I know this was carry over from the previous pack code but given `create_issue_link()` returns a `Response`, I wonder if we should rename the `issue` var to `response` to make that more clear.